src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/DefaultCaseDataAccessControl.java

```diff
@@ -52,6 +59,10 @@ public DefaultCaseDataAccessControl(RoleAssignmentService roleAssignmentService,
 
         RoleAssignmentFilteringResult filteringResults = roleAssignmentsFilteringService
             .filter(roleAssignments, caseDetails);
+        CaseTypeDefinition caseTypeDefinition = caseTypeService.getCaseType(caseDetails.getCaseTypeId());
+
+        List<AccessProfile> accessProfiles = roleAssignmentToAccessProfileMapper
```

Comment:
this should come after the step below, i.e. after the generation of fake role assignments